example/jobflow/README.md

```diff
@@ -0,0 +1,63 @@
+## JobFlow
+
+#### These examples shows how to run JobFlow via Volcano.
+
+[JobFlow](../../docs/design/jobflow) is a workflow engine based on volcano Job. It proposes two concepts to automate running multiple batch jobs, named JobTemplate and JobFlow, so end users can easily declare their jobs and run them using complex control primitives such as sequential or parallel execution, if-then -else statement, switch-case statement, loop execution, etc.
+
+read design at [here](../../docs/design/jobflow).
+
+### Prerequisites
+
+- docker: `18.06`
+- Kubernetes: >`1.17`
+
+## startup steps
+
+build image from local
+```bash
+# get volcano and jobflow source code from github
+git clone http://github.com/volcano-sh/volcano.git
+git clone https://github.com/BoCloud/JobFlow.git
```

Comment:
jobflow has been donated to the volcano community to become part of the volcano project. The relevant PR is under review.
https://github.com/volcano-sh/volcano/pull/2383
I don't think it's the right time to change this document at the moment, and wait until the jobflow code is fully integrated before optimizing the documentation.

/hold

Reply:
so..., no need for example/jobflow/README.md? what about README.md, if no-need change, i will close this pr

Reply:
This pr can stay open.I think it needs some modification, after that we can merge this pr, and after jobflow is merged, it may need to be modified again.

Reply:
@hwdef Good suggestion:). We can provide this example doc for our users and update it right after #2383 get merged.

Reply:
ok.